src/sos/__main__.py

```diff
@@ -1156,11 +1153,9 @@ def preview_file(previewers, filename, style=None):
             "metadata": {},
             "data": {
                 "text/plain":
-                    "\n> {} ({}):".format(
-                        filename, pretty_size(os.path.getsize(filename))),
+                    f"\n> {filename} ({pretty_size(os.path.getsize(filename))}):",
                 "text/html":
-                    HTML('<div class="sos_hint">> {} ({}):</div>'.format(
-                        filename, pretty_size(os.path.getsize(filename)))).data,
+                    HTML(f'<div class="sos_hint">> {filename} ({pretty_size(os.path.getsize(filename))}').data,
```

Comment:
Missing `</div>`.

Reply:
I reverted f'<div back to '<div. These changes can be found in the most recent commit.